docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+# This docker-compose file should be used for tests
```

Comment:
Should this file be moved into `unpub_aws` directory?

Reply:
fixed